portfolio/src/main/webapp/index.html

```diff
@@ -2,20 +2,18 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="indexstyle.css">
```

Comment:
index_style

Reply:
It looks like there are two versions of the CSS and HTML files. One with _ and one without. Can you remove unwanted ones.